<?xml version="1.0" ?>
<launch>

  <node pkg="mir_vision" type="detection_client" name="detection_client" output="screen" > 
    <param name="path_searchposes"          type="str"    value="$(find mir_vision)/config/searchPoses.config" /> 
    <param name="object_name"               type="str"    value="Goesser1" />
    <!-- <param name="learning_data"             type="str"    value="Teabox0_learning_data.xml" /> -->
    <param name="learning_data"             type="str"    value="Goesser17_learning_data.bin" />
    <param name="debug"                     type="bool"   value="true" /> 
    <!-- <param name="max_time"                  type="double" value="200" />  -->
  </node>

</launch>
